src/script_chainer/win_exe/script_runner.py

```
@@ -361,6 +384,75 @@ def run_script(
+
+        if log_notifier is not None:
+            sys.stdout = _TeeWriter(old_stdout, log_notifier)
+
+        exec_globals = {
+            '__name__': '__main__',
+            '__file__': script_path,
+            '__package__': None,
+            '__spec__': None,
+            '__builtins__': __builtins__,
+        }
+        exec(compile(code, script_path, 'exec'), exec_globals)
```

Ruff (0.15.10)

[error] Use of `exec` detected

(S102)